lms/static/js/search/views/search_list_view.js

```diff
@@ -57,7 +57,7 @@ define([
                 var item = new SearchItemView({ model: result });
                 return item.render().el;
             });
-            this.$el.find('.search-results').append(items);
+            this.$el.find('.search-results').html(items);
```

Comment:
Are we sure that we want to make this change? This seemingly has nothing to do with analytics stuff.